usr/lib/common/key.c

```diff
@@ -3388,13 +3518,14 @@ CK_RV ibm_dilithium_priv_check_required_attributes(TEMPLATE *tmpl, CK_ULONG mode
     CK_ULONG i;
 
     /* MODE_KEYGEN: attrs are added during keygen */
-    if (mode == MODE_KEYGEN)
+    if (mode == MODE_KEYGEN || mode == MODE_UNWRAP)
         return priv_key_check_required_attributes(tmpl, mode);
 
     /* MODE_CREATE (key import) or MODE_COPY: check if all attrs present */
     for (i = 0; i < sizeof(req_attrs) / sizeof(req_attrs[0]); i++) {
         if (!(template_attribute_find(tmpl, req_attrs[i], &attr))) {
-            TRACE_ERROR("%s\n", ock_err(ERR_TEMPLATE_INCOMPLETE));
+            TRACE_ERROR("%s, attribute at index %ld missing.\n",
```

Comment:
Better trace req_attr[i].type instead of the index.

Reply:
Fixed.